apps/system/init/init.c

```diff
@@ -63,6 +63,9 @@ extern void iotjs_register_cmds(void);
 #ifdef CONFIG_TASH
 static void tash_register_cmds(void)
 {
+	/* Register the basic commands first to always have "help" cmd */
+	tash_register_basic_cmds();
```

Comment:
Let's call tash_init before calling tash cmds instead of this change.

Reply:
Ok. Can be done.

Reply:
Oh, I am so sorry, I mean I want moving tash_start, not init. That's my fault.
current
```
int preapp_start()
{
tash_register_cmds();
tash_start();
}
```
suggestion
```
int preapp_start()
{
tash_start();
tash_register_cmds();
}
```